XSLT/modules/functions_module.xslt

```diff
@@ -325,18 +325,17 @@
     <xd:return>A fully-qualified URI, if the process succeeds, or the original
     string if it isn't possible to resolve the URI with the resources available.</xd:return>
   </xd:doc>
-  <xsl:function name="atop:resolve-private-uri" as="xs:string">
-    <xsl:param name="pUri" as="xs:string"/>
-    <xsl:param name="pContext" as="node()"/>
-
+  <xsl:function name="atop:resolve-uri" as="xs:anyURI">
+    <xsl:param name="pUri" as="xs:anyURI"/>
+    <xsl:param name="pContext" as="node()?"/>
     <xsl:choose>
       <xsl:when test="starts-with($pUri, 'tei:')">
         <xsl:if test="not(matches($pUri, '^tei:(current|[0-9]+\.[0-9]+\.[0-9])$'))">
           <xsl:message terminate="yes" expand-text="yes" error-code="atop:error-invalidOrMalformedURI">Invalid or malformed tei: private URI: '{$pUri}'</xsl:message>
```

Comment:
I realize it has nothing to do with this PR, but I would prefer a more verbose wording that does not risk reader coming to a mental stop at first colon. How about “Invalid or malformed private URI using the "tei:" scheme: …”?